velox/exec/tests/TableWriteTest.cpp

```diff
@@ -786,6 +786,18 @@ class TableWriteTest : public HiveConnectorTestBase {
     for (const auto bucketId : bucketIds) {
       ASSERT_EQ(expectedBucketId, bucketId);
     }
+
+    // Verifies the sorting behavior
+    if (testParam_.bucketSort()) {
+      auto sortedColumn1 = resultVector->childAt(4)->asFlatVector<double>();
```

Comment:
Let's store sorted column indexes as sortColumnIndices_ in TableWriteTest in Setup() as numPartitionKeyValues_ does.